### PR TITLE
Create slider feature

### PR DESCRIPTION
**Pull request by jaimelr**
I created a slider with pure javascript and exported the section to a schema so that the store owner can modify the image and text displayed in every slide.

slider

The development store:
[Fiorella theme](https://fiorella-dev-2.myshopify.com/)

### CODE REVIEW

File: fiorella/src/styles/main.scss [hunk 1]
```diff
@@ -4,11 +4,9 @@
     - You cannot use native CSS/Sass @imports in this file without a build script
 ==============================================================================*/
 
-/*================ UTILS ================*/
-@import url('tools/mixins.scss');
-
-/*================ SETTINGS ================*/
-@import url('settings/variables.scss.liquid');
+/*================ ABSTRACTS ================*/
```
Comment (magmahound) [inline fiorella/src/styles/main.scss]: Use `//` comments everywhere

File: fiorella/src/styles/layout/slider.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+@media #{$large-desktop-view} {
+  .slider-navigator {
+    bottom: 40px;
+  }
+
+  .slide-button {
+    bottom: 60px;
+    margin-right: 10px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered bottom, font-size, margin-right, padding

File: fiorella/src/styles/layout/slider.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+@media #{$large-desktop-view} {
+  .slider-navigator {
+    bottom: 30px;
+  }
+
+  .slide-button {
+    bottom: 50px;
+    margin-right: 10px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered bottom, font-size, margin-right, padding

File: fiorella/src/styles/layout/slider.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+@media #{$small-desktop-view} {
+  .slider-navigator {
+    bottom: 30px;
+  }
+
+  .slide-button {
+    bottom: 50px;
+    margin-right: 10px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered bottom, font-size, margin-right, padding

File: fiorella/src/styles/layout/slider.scss [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    display: flex;
+    justify-content: center;
+    position: absolute;
+    top: auto;
+    width: 100%;
+  }
+
+  .slide-button {
+    bottom: 40px;
+    padding: 10px 40px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered bottom, font-size, padding

File: fiorella/src/styles/layout/slider.scss [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+  font-style: italic;
+  font-size: 15px;
+  position: absolute;
+  right: 15px;
+  text-align: right;
+  width: 150px;
+}
+
+.slide-button {
+  display: inline;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered background-color, border-color, border-style, bottom, color, display, font-family, font-weight, padding, position, right, text-align, text-decoration, text-transform

File: fiorella/src/styles/layout/slider.scss [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+  img {
+    width: 100%;
+  }
+}
+
+.slide-description {
+  bottom: 60px;
+  font-family: $font-slider-description;
+  font-style: italic;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered bottom, font-family, font-size, font-style, position, right, text-align, width

File: fiorella/src/styles/layout/slider.scss [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+  top: 10px;
+  width: 100%;
+}
+
+.slider-nav-button {
+  background-color: $color-white-solid;
+  border-radius: 50%;
+  border-style: none;
+  font-family: sans-serif;
+  margin: 0 5px;
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Properties should be ordered background-color, border-radius, border-style, font-family, height, margin, width

File: fiorella/src/styles/layout/slider.scss [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+  font-family: $font-slider-description;
+  font-style: italic;
+  font-size: 15px;
+  position: absolute;
+  right: 15px;
+  text-align: right;
+  width: 150px;
+}
+
+.slide-button {
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Rule set contains (14/10) properties

File: fiorella/src/styles/layout/slider.scss [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+
+.slider-nav-button-selected {
+  background-color: $color-featured;
+}
+
+.show-slide {
+  display: flex;
+}
+
+.slide {
```
Comment (magmahound) [inline fiorella/src/styles/layout/slider.scss]: Merge rule `.slide` with rule on line 1

File: fiorella/src/styles/layout/slider.scss [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+  width: 100%;
+}
+
+.slider-nav-button {
+  background-color: $color-white-solid;
+  border-radius: 50%;
+  border-style: none;
+  font-family: sans-serif;
+  margin: 0 5px;
+  height: 12px;
```
Comment (IsmaelVillavicencio) [inline fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

File: fiorella/src/styles/layout/slider.scss (hunk 7 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

File: fiorella/src/styles/layout/slider.scss (hunk 5 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

File: fiorella/src/styles/layout/slider.scss (hunk 4 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

File: fiorella/src/styles/layout/slider.scss (hunk 3 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

File: fiorella/src/styles/layout/slider.scss (hunk 2 above)
Comment (IsmaelVillavicencio) [reply fiorella/src/styles/layout/slider.scss]: Please check magmahount's comments

**Pull request merged**